inst/json/graph_prevalence_config.json

```diff
@@ -48,20 +48,29 @@
             "id": "irs",
             "name": "IRS only",
             "type": "lines",
+            "line": {
+                "dash": "dot"
```

Comment:
also just changing these lines styles to match the original prototype - often 2 series are identical so it becomes useful to have variable line styles to make it more obvious where 2 or more series are overlayed.